### DISCUSSION TITLE
AttributeError: module 'pytorch_lightning' has no attribute 'metrics'

### DISCUSSION THREAD

**Discussion by NeighborhoodCoding** [Lightning-AI/pytorch-lightning#11005] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
AttributeError: module 'pytorch_lightning' has no attribute 'metrics'

what is this...? T.T

**Comment by ananthsub** [comment, marked as answer]
Pytorch Lightning Metrics were moved to a separate package/library/repo, torchmetrics, starting from Lightning 1.5

**Reply by rohitgr7** [reply]
adding link for it: https://github.com/PyTorchLightning/metrics
